google/_modules/gke/cluster.tf

```diff
@@ -48,6 +48,10 @@ resource "google_container_cluster" "current" {
     network_policy_config {
       disabled = false
     }
+
+    gcs_fuse_csi_driver_config {
```

Comment:
```
╷
│ Error: Unsupported block type
│ 
│   on ../google/_modules/gke/cluster.tf line 52, in resource "google_container_cluster" "current":
│   52:     gcs_fuse_csi_driver_config {
│ 
│ Blocks of type "gcs_fuse_csi_driver_config" are not expected here.
╵
```

This is probably an attribute that's only in the Google beta provider. I can't merge this into upstream because I can't force all users to use the beta provider by default. I can only merge this once the attribute is in the standard provider. Until then, to use this you'd have to point the module for your cluster to use your branch as the source and somehow use the google-beta provider.

I'm not sure how straight forward it would be to use the beta provider: https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/guides/provider_versions

Reply:
GCP support is working on promotion of this from `google-beta` to `google` provider.

Reply:
[gcs_fuse_csi_driver_config](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#gcs_fuse_csi_driver_config) is available in `google` from 4.76.0.